monkey/agent_plugins/exploiters/powershell/src/plugin.py

```diff
@@ -32,7 +28,8 @@
 
 
 def should_attempt_exploit(host: TargetHost) -> bool:
-    return not all_tcp_ports_are_closed(host, [POWERSHELL_NO_SSL_PORT, POWERSHELL_SSL_PORT])
+    closed_tcp_ports = host.ports_status.tcp_ports.closed
+    return not all([p in closed_tcp_ports for p in [POWERSHELL_NO_SSL_PORT, POWERSHELL_SSL_PORT]])
```

Comment:
alternative:
```suggestion
    return len(POWERSHELL_PORTS - host.ports_status.tcp_ports.closed) > 0
```

^ Assumes we create a new set called `POWERSHELL_PORTS`. 🤷

Reply:
Alternative:
```suggestion
    return POWERSHELL_PORTS.issubset(host.ports_status.tcp_ports.closed)
```

^ I think I like this one best. Thoughts?

Reply:
`<=` is syntactically equivalent to `issubset()`, but may be less clear.

```suggestion
    return POWERSHELL_PORTS <= host.ports_status.tcp_ports.closed
```

Reply:
I like `issubset` best as that is the only one that is clear enough. I will rework them using `issubset`.